tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -22,6 +22,44 @@
             networkAttachments:
             - storage
     '
+  when: glance_backend == 'local'
+
+- name: deploy podified Glance with Ceph backend
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      glance:
+        enabled: true
+        template:
+          databaseInstance: openstack
+          containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+          customServiceConfig: |
+            [DEFAULT]
+            enabled_backends=default_backend:rbd
+            [glance_store]
+            default_backend=default_backend
+            [default_backend]
+            rbd_store_ceph_conf=/etc/ceph/ceph.conf
+            rbd_store_user=openstack
+            rbd_store_pool=images
+            store_description=Ceph glance store backend.
+          storageClass: "local-storage"
+          storageRequest: 10G
+          glanceAPIInternal:
+            externalEndpoints:
```

Comment:
Looks good,  at some point we need to update this part according to [1], which is something applied to all the other operators as well. For now I think this looks very good and it's consistent with what we did in the doc. 

[1] https://github.com/openstack-k8s-operators/glance-operator/pull/285

Reply:
Tracking issue https://github.com/openstack-k8s-operators/data-plane-adoption/issues/124